lang_python.adoc

```diff
@@ -40,72 +48,80 @@ methods:
 
 * On Debian / Ubuntu, one can install it from apt repositories:
 
-** For Python 2.x:
+** For Python 3.x:
 +
 [source,shell]
-sudo apt-get install python-kaitaistruct
+sudo apt-get install -y python3-kaitaistruct
 +
-Reference: https://packages.debian.org/buster/python-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/testing/python3-kaitaistruct[image:https://repology.org/badge/version-for-repo/debian_testing/python:python-kaitaistruct.svg[package info for Debian]], https://packages.ubuntu.com/eoan/python3-kaitaistruct[image:https://repology.org/badge/version-for-repo/ubuntu_19_10/python:python-kaitaistruct.svg[package info for Ubuntu]]
 
-** For Python 3.x:
+** For Python 2.x:
 +
 [source,shell]
-sudo apt-get install python3-kaitaistruct
+sudo apt-get install -y python-kaitaistruct
 +
-Reference: https://packages.debian.org/buster/python3-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python3-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/stable/python-kaitaistruct[image:https://repology.org/badge/version-for-repo/debian_stable/python:python-kaitaistruct.svg[package info for Debian]], https://packages.ubuntu.com/eoan/python-kaitaistruct[image:https://repology.org/badge/version-for-repo/ubuntu_19_10/python:python-kaitaistruct.svg[package info for Ubuntu]]
+
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* Add the following dependency specifier into the package configuration
+of the packaging tool you use:
 +
 ----
 kaitaistruct
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+and then run `python3 -m pip install --upgrade -e ./` in the dir with your project
+in order to install your package and all its dependencies in
+an editable way. If you don't want to package your software,
+but only installbits dependencies, add `kaitaistruct` into `requirements.txt`.
 
-* Otherwise, you can just install the package manually using *pip*
+* Otherwise, you can just install the package manually using `pip`
   (Python package manager):
 +
 [source,shell]
-pip install kaitaistruct
+python3 -m pip install --upgrade kaitaistruct
 
 [[add-runtime-snapshot]]
 ==== Using latest (snapshot) version
 
-If you're using latest (unreleased AKA "unstable" AKA "snapshot") build
+If you're using latest (unreleased AKA "unstable" AKA "snapshot" AKA "nightly") build
 of Kaitai Struct compiler, that will require latest runtime libraries as
 well:
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* You can a make `pip` to use a bleeding edge version of the runtime by
```

Comment:
_"Bleeding edge"_ seems a bit tabloid to me, what about "development version" or "the most recent" version?

Reply:
`version, incorporating most recent time-untested changes`?

Reply:
> version, incorporating the most recent time-untested changes

This phrase can be unambiguously replaced with a widely accepted term _unstable_.

Reply:
`unstable` has ambigious meaning:

1. changing frequently in incompatible ways.
2. crashing constantly.

I guess the second one doesn't apply here.

Reply:
What about the development version?

Reply:
A bit better, but doesn't reflect the fact it can contain undiscovered and undocumented bugs with a bit higher probability.

Reply:
OK, feel free to keep it, it's not so important.

Reply:
used the term `snapshot`